### ISSUE TITLE
Datafile::flush doesn't do anything

### ISSUE THREAD

**Issue by d7919** [boutproject/BOUT-dev#374]
From what I can see the datafile::flush parameter is never used for anything. Is this correct or am I missing something? The `openclose` parameter seems to be do something similar to a flush (by closing and then reopening the file), has this replaced `flush` or is flush intended to be used for something else yet to be implemented?

**Comment by bendudson** [comment]
Agreed, this doesn't seem to do anything now. In the header the comment is "Flush after every write?" but I guess this is now replaced by openclose. Perhaps it was used for some old formats (PDB?)

**Comment by d7919** [comment]
OK thanks, I couldn't find any use of it in master either, but maybe it was used historically. Is it worth removing references to this?

**Comment by bendudson** [comment]
I think datafile and friends need to be sorted out after the Great Merge, so this can probably wait

**Comment by d7919** [comment]
OK, I'll leave this issue open as a reminder.
